server/src/main/java/com/broadcom/lsp/cobol/service/delegates/communications/ServerCommunications.java

```diff
@@ -83,7 +87,7 @@ public void notifyThatEngineNotFound(String languageType) {
   public void notifyThatLoadingInProgress(String uri) {
     String decodedUri = files.decodeURI(uri);
     uriInProgress.add(decodedUri);
-    executor.schedule(
+    customExecutor.getScheduledThreadPoolExecutor().schedule(
```

Comment:
Will schedule each task in a new executor

Reply:
> Will schedule each task in a new executor

done